FantasyUniverse.xcodeproj/project.pbxproj

```diff
@@ -561,7 +593,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ -f \"${PROJECT_DIR}/SwiftGen/bin/swiftgen\" ]]; then\n\necho $SRCROOT\n\n  rm \"$SRCROOT/FantasyUniverse/Generated/Strings.swift\"\n\n  rm \"$SRCROOT/FantasyUniverse/Generated/Assets.swift\"\n\n  \"${PROJECT_DIR}/SwiftGen/bin/swiftgen\" config run\n\nelse\n\n  echo \"warning: SwiftGen is not installed. Run 'pod install --repo-update' to install it.\"\n\nfi\n";
+			shellScript = "if [ $ACTION != \"indexbuild\" ]; then\nif [[ -f \"${PROJECT_DIR}/SwiftGen/bin/swiftgen\" ]]; then\n\necho $SRCROOT\n\n  rm \"$SRCROOT/FantasyUniverse/Generated/Strings.swift\"\n\n  rm \"$SRCROOT/FantasyUniverse/Generated/Assets.swift\"\n\n  \"${PROJECT_DIR}/SwiftGen/bin/swiftgen\" config run\n\nelse\n\n  echo \"warning: SwiftGen is not installed. Run 'pod install --repo-update' to install it.\"\n\nfi\nfi\n\n";
```

Comment:
That's interesting, why is this change needed?

Reply:
Hm.. as far as I remember it fixed some swiftgen code generating issues.